conf.d/no-ssl.default.conf

```diff
@@ -6,8 +6,8 @@
 #
 # If no default server is defined, Nginx will use the first found server.
 # To prevent host header attacks, or other potential problems when an unknown
-# servername is used in a request, it's recommended to drop the request
-# returning 444 "no response".
+# `server_name` is used in a request, it's recommended to drop the request
```

Comment:
Nope, this refers to a variable, actually `$hostname` (or whichever used for server selection).
Maybe add a space for correctness, but let keep "server name" reference for "comprehensibility".

```suggestion
# server name is used in a request, it's recommended to drop the request
```

Reply:
Changed in https://github.com/h5bp/server-configs-nginx/pull/232/commits/086207053847e6d952981a77659135b9d4f324e0